sites/fast-website/src/app/components/content-placement-container/content-placement-container.styles.ts

```diff
@@ -23,4 +117,14 @@ export const ContentPlacementContainerStyles = css`
         filter: blur(0);
     }
     /* end */
+
+    .icon {
+        fill: currentColor;
+    }
+
+    h3 {
+        margin-top: 0;
```

Comment:
these can be combined into the shorthand

Reply:
Thank you for the catch. I will update it.